compiler/rustc_codegen_ssa/src/back/metadata.rs

```diff
@@ -199,7 +199,7 @@ fn create_object_file(sess: &Session) -> Option<write::Object<'static>> {
 // * ELF - All other targets are similar to Windows in that there's a
 //   `SHF_EXCLUDE` flag we can set on sections in an object file to get
 //   automatically removed from the final output.
-pub fn create_rmeta_file(sess: &Session, metadata: &[u8]) -> Vec<u8> {
+pub fn create_rmeta_file(sess: &Session, metadata: &[u8]) -> (Vec<u8>, bool) {
```

Comment:
Nit: would it make sense to replace this `bool` with an enum? It isn't entirely obvious at a glance what this represents and also it seems like it'd be quite straightforward to accidentally ignore it, even though the data is quite different depending on the branch taken.

Reply:
Done